perses/app/relative_point_mutation_setup.py

```diff
@@ -1,75 +1,59 @@
-from scipy.special import logsumexp
-from pkg_resources import resource_filename
 from perses.tests.utils import validate_endstate_energies
+from openforcefield.topology import Molecule
+from openmmforcefields.generators import SystemGenerator
 
 ENERGY_THRESHOLD = 1e-2
 temperature = 300 * unit.kelvin
 kT = kB * temperature
 beta = 1.0/kT
 
-
-
 class PointMutationExecutor(object):
     """
     Simple, stripped-down class to create a protein-ligand system and allow a mutation of a protein.
     this will allow support for the creation of _two_ relative free energy calculations:
         1. 'wildtype' - 'point mutant' complex hybrid.
-        2. 'wildtype' - 'point mutant' receptor hybrid (i.e. with ligand of interest unbound)
+        2. 'wildtype' - 'point mutant' protein hybrid (i.e. with ligand of interest unbound)
 
     Example (create full point mutation executor and run parallel tempering on both complex and apo phases):
-        receptor_path = 'data/perses_jacs_systems/thrombin/Thrombin_protein.pdb'
+        from pkg_resources import resource_filename
+
+        protein_path = 'data/perses_jacs_systems/thrombin/Thrombin_protein.pdb'
         ligands_path = 'data/perses_jacs_systems/thrombin/Thrombin_ligands.sdf'
-        receptor_filename = resource_filename('openmmforcefields', receptor_path)
+        protein_filename = resource_filename('openmmforcefields', protein_path)
         ligand_filename = resource_filename('openmmforcefields', ligands_path)
 
-        pm_delivery = PointMutationExecutor(receptor_filename = receptor_filename,
-                                    ligand_filename = ligand_filename,
-                                    mutation_chain_id = '2',
-                                    mutation_residue_id = '198',
-                                     proposed_residue = 'THR',
-                                     phase = 'complex',
-                                     conduct_endstate_validation = False,
-                                     ligand_index = 0,
-                                     forcefield_files = ['amber14/protein.ff14SB.xml', 'amber14/tip3p.xml'],
-                                     barostat = openmm.MonteCarloBarostat(1.0 * unit.atmosphere, temperature, 50),
-                                     forcefield_kwargs = {'removeCMMotion': False, 'ewaldErrorTolerance': 1e-4, 'nonbondedMethod': app.PME, 'constraints' : app.HBonds, 'hydrogenMass' : 4 * unit.amus},
-                                     small_molecule_forcefields = 'gaff-2.11')
+        pm_delivery = PointMutationExecutor(protein_filename=protein_filename,
+                                    mutation_chain_id='2',
+                                    mutation_residue_id='198',
+                                     proposed_residue='THR',
+                                     phase='complex',
+                                     conduct_endstate_validation=False,
+                                     ligand_filename=ligand_filename,
```

Comment:
just document in the `arguments` that `.pdb` reads as protein whereas `.sdf` reads as small molecule